src/output/icons.rs

```diff
@@ -302,6 +302,9 @@ pub fn icon_for_file(file: &File<'_>) -> char {
             "svg"           => '\u{f1c5}', // 
             "swift"         => '\u{e755}', // 
             "tar"           => '\u{f410}', // 
+            "tar.gz"        => '\u{f410}', // 
+            "tar.lz"        => '\u{f410}', // 
+            "tar.xz"        => '\u{f410}', // 
```

Comment:
I think "tgz", "tlz", "txz" should be added as well :)

Reply:
They should added *instead*. Here, only the part after the last dot is matched against, and gz, lz and xz are already in the list.

Reply:
Hi @ariasuni I think @jim4067 forgot to mention that he already addressed your comment in commit 437cd5b7f40ba1b704de211300b2eed42c93e71b